catalog.html

```diff
@@ -0,0 +1,134 @@
+  <body>
+    <header>
+      <nav>
+        <a href="blank.html"><img src="https://via.placeholder.com/150x100" alt="catalog"></a>          
+        <ul>
+          <li><a href="blank.html">Информация</a></li>
+          <li><a href="blank.html">Фото и видео</a></li>
+          <li><a href="blank.html">Карта штата</a></li>
+          <li><a href="catalog.html">Гостиницы</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+        <form action="https://echo.htmlacademy.ru/courses" method="post">
+            <ul>
+                <li>
+                    <h2>ИНФРАСТРУКТУРА:</h2>
+                </li>
+                <li>
+                    <input type="checkbox" name="swimmingpool" id="swimmingpool-field" checked>
+                    <label for="swimmingpool-field">БАССЕЙН</label>
+                </li>
+                <li>
+                    <input type="checkbox" name="parking" id="parking-field">
+                    <label for="parking-field">ПАРКОВКА</label>
+                </li>
+                <li>
+                    <input type="checkbox" name="wi-fi" id="wi-fi-field">
+                    <label for="wi-fi-field">WI-FI</label>
+                </li>
+            </ul>
+            <ul>
+                <li>
+                    <h2>ТИП ЖИЛЬЯ:</h2>
+                </li>
+                <li>
+                    <input type="checkbox" name="hostel" id="hosltel-field" checked>
+                    <label for="hostel-field">ГОСТИНИЦА</label>
+                </li>
+                <li>
+                    <input type="checkbox" name="motel" id="motel-field" checked>
+                    <label for="motel-field">МОТЕЛЬ</label>
+                </li>
+                <li>
+                    <input type="checkbox" name="apartments" id="apartments-field" checked>
+                    <label for="apartments-field">АПАРТАМЕНТЫ</label>
+                </li>
+            </ul>
+            <p>
+                <label for="cost">СТОИМОСТЬ В СУТКИ, ₽: </label>
+                <button type="button">ОТ 0</button>
+                <button type="button">ДО 3000</button>
+            </p>
+            <button type="submit">ПОКАЗАТЬ</button>
+        </form>
+        <section>
+            <h2>НАЙДЕНО:3</h2>
+            <p>СОРТИРОВКА</p>
+            <p>ПО ЦЕНЕ</p>
+            <p>ПО ТИПУ</p>
+            <p>ПО РЕЙТИНГУ</p>
+            <button type="button">
+                <img src="https://via.placeholder.com/150x100" alt="more"> 
+            </button> 
+            <button type="button">
+                <img src="https://via.placeholder.com/150x100" alt="less"> 
+            </button> 
+            <ul>
+                <li>
+                  <h3>AMARA RESORT & SPA</h3>
+                  <p>ГОСТИНИЦА</p>
+                  <p>ОТ 4000 ₽</p>
+                  <button type="button"><a href="blank.html">ПОДРОБНЕЕ</a></button>
+                  <button type="button"><a href="blank.html">ЗАБРОНИРОВАТЬ</a></button>
```

Comment:
Ссылка не может быть внутри кнопки. Данные элементы интерфейса размечаются ссылками:
```suggestion
                  <a href="blank.html">ПОДРОБНЕЕ</a>
                  <a href="blank.html">ЗАБРОНИРОВАТЬ</a>
```